index.html

```diff
@@ -0,0 +1,218 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title></title>
+	<link rel="stylesheet" href="quiz.css">
+	<link href='https://fonts.googleapis.com/css?family=Raleway:300,400,600' rel='stylesheet' type='text/css'>
+</head>
+<body>
+	<nav>
+		<div class="title">
+			<h1>Downtown Guitars</h1>
+			<h3>Where guitars are downtown...</h3>
+		</div>
+	</nav>
+
+	<article class="jazzmaster">
+		<header>
+			<h1>Fender Jazzmaster</h1>
+		</header>
+
+		<section>
+			<img src="../Jazzmaster-500x500.jpg" alt="Fender Jazzmaster">
```

Comment:
The image is technically centered, but it isn't inside a block element (Requirement 5-i). the 20% margin will bite you at lower screen sizes.